src/main/java/org/geysermc/generator/MapColors.java

```diff
@@ -0,0 +1,67 @@
+package org.geysermc.generator;
+
+import java.awt.*;
```

Comment:
I assume this is only importing the Color class? Then there's no need for this to be a wildcard.